backend/src/main/services/LogHoursService.ts

```diff
@@ -71,10 +71,18 @@ export class LogHoursService {
     return this.payRepository.create(payCreationDTO);
   };
 
+  public getAllBusinessPays = async (businessId: number): Promise<Pay[] | null> => {
```

Comment:
Can you also add the backend tests for these new log hour services services?

Reply:
didn't we decide it wasn't worth testing getters since we would just be mocking and returning the mocked value immediately?